test/e2e_flags.go

```diff
@@ -58,6 +59,16 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 		f.ResolvableDomain = fl.Value.(flag.Getter).Get().(bool)
 	}
 
+	if fl := flag.Lookup("clusterSuffix"); fl == nil {
+		// Only define and set flags here. Flag values cannot be read at package init time.
```

Comment:
super-nit, but looks like this comment got copy-pasted from above so is now duplicated here, I don't think we need it three times :).